src/utils/converters.ts

```diff
@@ -0,0 +1,30 @@
+import firebase from "firebase/app";
+
+import { AuditoriumSeatedUser } from "types/auditorium";
+import { AnyVenue } from "types/venues";
+
+import { WithId, withId } from "utils/id";
+
+export const venueConverter: firebase.firestore.FirestoreDataConverter<
```

Comment:
That's a really useful concept here👍
Can we make generic `withIdConverter` that to deduplicate `venueConverter` and `auditoriumSeatedUserConverter`?

Reply:
I think we can and I tried to, but I need to play with types a bit